cmd/flagsaver/flagsaver.go

```diff
@@ -0,0 +1,36 @@
+// Package flagsaver provides a simple way to save and restore flag values.
+// TODO(RJPercival): Move this to its own GitHub project.
+//
+// Example:
+//   func TestFoo(t *testing.T) {
+//     defer flagsaver.Save().Restore()
+//     // Test code that changes flags
+//   } // flags are reset to their original values here.
+package flagsaver
```

Comment:
flagsaver isn't a command we run, so it shouldn't be placed inside cmd/. It definitely deserves its own package, though. Maybe util/flagsaver?